src/canmatrix/formats/arxml.py

```diff
@@ -927,9 +927,12 @@ def decode_compu_method(compu_method, root_or_cache, ns, float_factory):
             numerator = get_children(numerator_parent, "V", root_or_cache, ns)
             denominator_parent = get_child(rational, "COMPU-DENOMINATOR", root_or_cache, ns)
             denominator = get_children(denominator_parent, "V", root_or_cache, ns)
-
-            factor = float_factory(numerator[1].text) / float_factory(denominator[0].text)
-            offset = float_factory(numerator[0].text) / float_factory(denominator[0].text)
+            try:
+                factor = float_factory(numerator[1].text) / float_factory(denominator[0].text)
+                offset = float_factory(numerator[0].text) / float_factory(denominator[0].text)
+            except decimal.DivisionByZero:
```

Comment:
This is due to example provided in #368, right? I believe that the "V"alues in the (DE)NOMINATOR are wrong, because they should represent coefficients of a polynomial function and it would be (0+x)/(0+x) = 1 regardless of the signal value (and undefined for for x=0). Strange.
I'd write at least an error message to the console, that the polynom is not supported and it is replaced by factor=1.

Reply:
@Funth0mas 
Thanks for this hint.
Fixed this with f29af5a